### ISSUE TITLE
Implement Fire TV version?

### ISSUE THREAD

**Issue by bbodenmiller** [OneBusAway/onebusaway-android#762]
Has anyone considered or looked in to making a [Fire TV / Fire TV stick version of OBA](https://developer.amazon.com/fire-tv)? Seems like if it could be pulled off it might be the cheapest way to get OBA running on TV screens in businesses, at stops, etc. Put a few up outside Amazon's offices in Seattle and they might even donate some of the Fire TV sticks.

Some challenges:
* [ ] Need large display stop mode
* [ ] Can it be remotely managed? - various Google searches for "Fire TV VNC" seem to indiacate a number of Android VNC apps work to a certain extent
* [ ] Can it be setup to automatically start the app on startup
* [ ] Fire TV stick has to use wireless, Fire TV could use a wired connection

**Comment by barbeau** [comment]
I think the web sign mode is a better fit for this:
https://github.com/OneBusAway/onebusaway-application-modules/wiki/Sign-Mode

**Comment by bbodenmiller** [comment]
Interesting, I was interested in exploring this for everyone but I guess not desired. Sign mode was used in Seattle but it was super unreliable and as of a few months ago most the signs are dead. The machines they were running on would die and the website itself didn't perform well. I think minor tweaks to the Android app could make it usable for sign mode.

**Comment by barbeau** [comment]
I'll leave this open if you're passionate about pursuing it, but I'm not sure I see the additional value over just pulling up sign mode from the browser for a Chromecast or Fire TV / stick.

**Comment by bbodenmiller** [comment]
Fire TV has a browser?

**Comment by barbeau** [comment]
I was thinking casting (https://productforums.google.com/forum/m/#!topic/chromecast/JZYePYUSABQ) or display mirroring (https://www.amazon.com/gp/help/customer/display.html?nodeId=201453020).  It's arguably more convenient to use a native app rather than mirrorong/casting, but given the market size I'm not sure it's worth the dev effort. But, PRs happily accepted :).

**Comment by bbodenmiller** [comment]
I agree the market is extremely small but I'll take a look. It may take some time though.

**Comment by bbodenmiller** [comment]
FYI there is an post yesterday about the signs at https://www.seattletransitblog.com/2017/02/20/better-oba-kiosks-coming/. There's a number of ideas for an improved sign mode as well feedback on OBA in general.
